src/obr/core/core.py

```diff
@@ -422,3 +422,19 @@ def is_time(s: str) -> bool:
     ret = [path / f for f in fs if is_time(f)]
     ret.sort()
     return ret
+
+
+def profile_call(f, *args, **kwargs):
+    if os.environ.get("OBR_PROFILE"):
+        logging.info("Generating profile")
+        import cProfile
+        import pstats
```

Comment:
How often is this called? If frequent, I would consider importing these once on a global level